core/domain/exp_services_test.py

```diff
@@ -774,8 +774,9 @@ def test_loading_exploration_from_yaml_does_not_override_existing_id(self):
         exp_services.load_demo(self.DEMO_EXP_ID)
 
         # Override the demo exploration using the import method.
-        exp_services.save_new_exploration_from_yaml_and_assets(
-            self.owner_id, self.SAMPLE_YAML_CONTENT, self.DEMO_EXP_ID, [])
+        with self.swap(feconf, 'ENABLE_STATE_ID_MAPPING', False):
```

Comment:
We can just delete this test altogether.

Reply:
Done